bluesky/callbacks/best_effort.py

```diff
@@ -161,7 +172,7 @@ def descriptor(self, doc):
         if stream_name == self.dim_stream:
             # Ensure that no independent variables ('dimensions') are
             # duplicated here.
-            columns = [c for c in columns if c not in self.dim_fields]
+            columns = [c for c in columns if c not in self.all_dim_fields]
 
             if self._table_enabled:
                 self._table = LiveTable(list(self.dim_fields) + columns)
```

Comment:
This should use `all_dim_fields` instead.

Reply:
good catch! livetable should plot everything